.github/workflows/release.yml

```
@@ -1,54 +1,60 @@
+        run: |
+          mkdir -p dist/keyflip-macos
+          go build -o dist/keyflip-macos/keyflip ./cmd/keyflip
+          cp layouts.json dist/keyflip-macos/
+
+      - name: Zip macOS
+        run: |
+          cd dist
+          zip -r keyflip-macos-${GITHUB_REF_NAME}.zip keyflip-macos
+
+      - name: Upload macOS Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)